backend/tests/configuration/environment.test.ts

```diff
@@ -0,0 +1,29 @@
+import { assert, assertEquals, assertExists, assertThrows } from 'std/testing/asserts.ts';
+import { DotenvConfig } from 'https://deno.land/x/dotenv@v3.2.0/mod.ts';
+import { envVarsSchema } from '../../src/configuration/environment.ts';
+
+Deno.test('if config validation works', () => {
+  const config: DotenvConfig = {
+    PORT: '8000',
+    HOSTNAME: 'localhost',
+    FRONTEND_DIR: 'frontend',
+  };
+  const { error, value: envVars } = envVarsSchema.validate(config);
+  assert(error === undefined);
+  assertExists(envVars);
+  assertEquals(envVars.PORT, 8000);
+  assertEquals(envVars.HOSTNAME, 'localhost');
+  assertEquals(envVars.FRONTEND_DIR, 'frontend');
+});
+
+Deno.test('if config validation error works', () => {
+  assertThrows(() => {
+    const config: DotenvConfig = {
+      PORT: '90000',
+      HOSTNAME: 'localhost',
+      FRONTEND_DIR: 'frontend',
+    };
```

Comment:
```suggestion
    const config: DotenvConfig = { ...defaultConfig, PORT: '90000' };
```
Part 2/2:
And use the spread operator here, so we do not have to modify two const just for the case that we are adding more and more .env vars